Sources/ClientRuntime/Signing/SigningAlgorithm+CRT.swift

```diff
@@ -4,18 +4,11 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 //
-import AwsCommonRuntimeKit
 
-/// Type of signing algorithm
-/// String raw value used for serialization and deserialization
-public enum AWSSigningAlgorithm: String {
-    ///  Signature Version 4
-    case sigv4
-    ///  Signature Version 4 Asymmetric
-    case sigv4a
-}
+import SmithyHTTPAuthAPI
+import AwsCommonRuntimeKit
 
-extension AWSSigningAlgorithm {
+extension SigningAlgorithm {
```

Comment:
Type was moved out to `SmithyHTTPAuthAPI` but this CRT-dependent extension remains.